test/lib/rand.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const assert = require('chai').assert;
+const rndid = require('../../lib/utils/rndid');
+
+describe('lib/utils/rndid', () => {
+  it('should generate a random id defaults to 8', () => {
```

Comment:
Maybe "should generate a random 8-character id" or "should generate a random id with a default length of 8 characters"? This could imply that the id defaults to `8`